### ISSUE TITLE
[BUG][WEBJS] - In WEBJS mode, the webhook is lost in group messages, and it needs to be logged out and started again to solve the problem.

### ISSUE THREAD

**Issue by HuangDaHui** [allburov/whatsapp-http-api#459]
I would like to ask if anyone has encountered the same problem? I didn't save the log and will need to provide log records next time

**Comment by devlikepro** [comment]
Yes, we've seen one time the same issue, but the general restart (stop and start the same session) helped in that case, not sure what causes your issue, waiting for the logs if you have any, it'll help!

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by HuangDaHui** [comment]
> 是的，我们曾经见过同样的问题，但常规重启（停止并启动相同的会话）在这种情况下有所帮助，不确定是什么导致了你的问题，如果有的话，等待日志，它会有所帮助！
> 
> [![赞助人：PRO](https://camo.githubusercontent.com/53c8c17e54974d30f0938bed9c2aa4e5c7dc7bc776296a6776840124e4086a04/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f706174726f6e2d50524f2d313838613432)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)



> 是的，我们曾经遇到过同样的问题，但经常重启（停止并启动相同的会话）在这种情况下会有所帮助，不确定是什么导致了你的问题，如果有的话，等待日志，它会有帮助！
> 
> [![赞助商：PRO](https://camo.githubusercontent.com/53c8c17e54974d30f0938bed9c2aa4e5c7dc7bc776296a6776840124e4086a04/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f706174726f6e2d50524f2d313838613432)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

Yes, I currently solve the problem by stopping and restarting. I suspect it has something to do with the server, but I haven’t found any useful logs yet.
